admin/controllers/check-form.js

```diff
@@ -22,10 +24,12 @@ const winston = require('winston')
  */
 const getTestDeveloperHomePage = async (req, res, next) => {
   res.locals.pageTitle = 'MTC for test development'
+  const isNewCheckForm = featureToggles.isFeatureEnabled('newCheckForm')
   try {
     req.breadcrumbs(res.locals.pageTitle)
     res.render('test-developer/test-developer-home', {
-      breadcrumbs: ''
+      breadcrumbs: '',
+      isNewCheckForm
```

Comment:
It's not clear from the name that this `isNewForm` is a feature toggle.  What do you think of `isNewCheckFormFeatureToggleEnabled`?  I am open to other suggestions.

Reply:
@jon-shipley updated as suggested.